test/e2e/providers/kind.go

```diff
@@ -18,13 +18,10 @@ import (
 	"fmt"
 	"os"
 	"path"
-
+	"strings"
 	"github.com/vmware-tanzu/antrea/test/e2e/providers/exec"
 )
 
-// TODO: try not to hardcode this name if possible.
-const masterNodeName = "kind-control-plane"
-
 type KindProvider struct{}
```

Comment:
you can add the `masterNodeName` as a field in `KindProvider`. This way you don't need to pass an extra parameter to `enableKubectlOnMaster` and `moveCoreDNSPodsToMaster`.